index.yaml

```diff
@@ -132,8 +132,19 @@ sources:
     homepage: https://raw-data.gitlab.io/post/malsilo_2.1/
     checksum: true
 
+  stamus/lateral:
+    summary: Lateral movement rules
+    description: |
+      Suricata ruleset specifically focused on detecting lateral
+      movement in Microsoft Windows environments by Stamus Networks
+    vendor: Stamus Networks
+    min-version: 6.0.6
+    license: GPLv3
+    support-url: https://discord.com/channels/911231224448712714/911238451842666546
+    url: https://ti.stamus-networks.io/open/stamus-lateral-rules.tar.gz
+    checksum: https://ti.stamus-networks.io/open/stamus-lateral-rules-checksum.txt
```

Comment:
This form of specifying a checksum URL does not work yet (https://redmine.openinfosecfoundation.org/issues/5684), @pevma, can you create the checksum URL at the rule URL + `.md5`?

Reply:
Fixed